doc/source/user_guide/mapdl.rst

```diff
@@ -665,83 +665,131 @@ additional commands that are incompatible with the way inputs are
 handled in the MAPDL server.
 
 
-Non-Applicable Commands
+.. _ref_unsupported_commands:
+
+Non-available Commands
 ~~~~~~~~~~~~~~~~~~~~~~~
-Some commands are quietly ignored by MAPDL and you are still free to
-use them.  For example ``/BATCH``, implemented as ``Mapdl.batch()`` returns:
+Some commands are not available in PyMAPDL because of different reasons.
+
+Some these commands do not make sense in a Python context.
+For example the ``*ASK`` can be replaced with a Python ``input``,
+``*IF`` with a Python ``if`` statement, and instead of ``*CREATE`` and
+``*USE`` can simply call another Python function or module.
```

Comment:
Provide a link to mapdl.input

Otherwise, great notes.

Reply:
But it is python input.

Reply:
Done.